pkg/test/framework/components/echo/config.go

```diff
@@ -251,6 +251,10 @@ func (c Config) IsDelta() bool {
 	return len(c.Subsets) > 0 && c.Subsets[0].Annotations != nil && strings.Contains(c.Subsets[0].Annotations.Get(SidecarProxyConfig), "ISTIO_DELTA_XDS")
 }
 
+func (c Config) IsRegularPod() bool {
```

Comment:
super nit: might be worth providing a comment on what is meant by "regular". It looks like it means "kubernetes pod holding a workload instance with an injected sidecar", but maybe it is easier to define it by what it is not ?

Reply:
Can either reference `SingleSimplePodServiceAndAllSpecial`'s comment, or move that comment here and have that method reference this one.